corehq/apps/users/models.py

```diff
@@ -1170,7 +1170,7 @@ def get_django_user(self, use_primary_db=False):
         queryset = User.objects
         if use_primary_db:
             queryset = queryset.using(router.db_for_write(User))
-        return queryset.get(username__iexact=self.username)
+        return queryset.get(username=self.username)
```

Comment:
Can you say a bit about how you determined that this change was safe?

Reply:
Take a look at https://github.com/dimagi/commcare-hq/pull/33824 and feel free to ask any questions there.